Index: app/src/main/java/com/example/project_start/User.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/project_start/User.java b/app/src/main/java/com/example/project_start/User.java
--- a/app/src/main/java/com/example/project_start/User.java	
+++ b/app/src/main/java/com/example/project_start/User.java	(date 1740389908123)
@@ -7,6 +7,7 @@
 import com.google.firebase.database.IgnoreExtraProperties;
 
 import java.io.ByteArrayOutputStream;
+import java.util.ArrayList;
 import java.util.Date;
 
 @IgnoreExtraProperties
@@ -16,7 +17,6 @@
     private String email,firstname, lastname;
     private Date date;
     private String pic;
-    private String preferredClub;
 
     public User(String uid, String email, String firstname,String lastname, Date date,String pic) {
         this.uid = uid;
@@ -86,12 +86,4 @@
     public void setUid(String uid) {
         this.uid = uid;
     }
-
-    public String getPreferredClub() {
-        return preferredClub;
-    }
-
-    public void setPreferredClub(String preferredClub) {
-        this.preferredClub = preferredClub;
-    }
 }
Index: app/src/main/java/com/example/project_start/My_Teams.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/project_start/My_Teams.java b/app/src/main/java/com/example/project_start/My_Teams.java
--- a/app/src/main/java/com/example/project_start/My_Teams.java	
+++ b/app/src/main/java/com/example/project_start/My_Teams.java	(date 1740478047571)
@@ -59,13 +59,14 @@
 
         firebaseDatabase = FirebaseDatabase.getInstance("https://newpcproject-c165b-default-rtdb.europe-west1.firebasedatabase.app/");
 
-        firebaseDatabase = FirebaseDatabase.getInstance();
         firebaseDatabase.getReference("Teams").addValueEventListener(new ValueEventListener() {
             @Override
             public void onDataChange(@NonNull DataSnapshot snapshot) {
+
                 for (DataSnapshot dataSnapshot : snapshot.getChildren()) {
                     Team team = dataSnapshot.getValue(Team.class);
                     teamsList.add(team);
+
                 }
                 Teams_Adapter adapter = new Teams_Adapter(My_Teams.this, teamsList);
                 rv.setAdapter(adapter);
